GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -136,6 +137,24 @@ public void setScript(GVRScript gvrScript, String distortionDataFileName) {
         }
     }
 
+    /**
+     * Sets whether to force rendering to be single-eye, monoscopic view.
+     *
+     * @param force
+     *            If true, will create a GVRMonoscopicViewManager when {@linkplain setScript setScript()} is called.  If false, will proceed to auto-detect whether the device supports VR rendering and choose the appropriate ViewManager.  This call will only an effect if it is called before {@linkplain setScript setScript()}.
+     *
+     */
+    public void setForceMonoscopic(boolean force) {
+        forceMonoscopic = force;
+    }
+
+    /**
+     * Returns whether a monoscopic view will be created during {@linkplain setScript setScript()}.
+     */
+    public boolean getForceMonoscopic() {
+        return forceMonoscopic;
+    }
```

Comment:
Actually I think the logic here is also incorrect. What you should return is forceMonoscopic || !isVrSupported()